partcad-cli/src/partcad_cli/click/commands/list/parts.py

```
@@ -23,20 +23,27 @@
         return
     package = package_obj.name
 
-    with Process("ListParts", package):
+    with pc_logging.Process("ListParts", package):
         part_count = 0
         part_kinds = 0
 
         if used_by is not None:
-            logging.info("Instantiating %s..." % used_by)
-            ctx.get_assembly(used_by)
+            pc_logging.info("Instantiating %s..." % used_by)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)